CHANGELOG.md

```diff
@@ -1,19 +1,21 @@
-## 2.10.0-beta.1
+## 2.10.0
 
 ## Features ✨ and improvements 🏁
+* Add new marker styling option rotationAlignment: 'horizon' allowing marker rotation to match the curvature of the horizon in globe view ([#11894](https://github.com/mapbox/mapbox-gl-js/pull/11894))
```

Comment:
Are we adding in new features that weren't in the beta, or were these just not highlighted previously in the changelog?

Reply:
Ah nevermind I see they are re-ordered! Thanks

Reply:
There were a few features missing from the previous changelog, otherwise they've been reordered.